File: guides/micronaut-webjars-stimulus/java/src/main/java/example/micronaut/HomeController.java
package example.micronaut;

import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;
import io.micronaut.views.View;

import java.util.Collections;
import java.util.Map;

@Controller // <1>
class HomeController {
    @Get // <2>
    @View("index.html") // <3>
    Map<String, Object> index() {
        return Collections.emptyMap();
    }
}

File: guides/micronaut-webjars-stimulus/python/src/example/micronaut/home_controller.py
from micronaut.http import MediaType
from micronaut.http.annotation import Controller, Get
from micronaut.views import View


@Controller  # <1>
class HomeController:

    @View("index.html")  # <3>
    @Get(produces=MediaType.TEXT_HTML)  # <2>
    def index(self) -> dict[str, object]:
        return {}
